emmet-core/emmet/core/structure_group.py

```diff
@@ -223,7 +223,7 @@ def group_entries_with_structure_matcher(
 def _get_id_num(task_id) -> Union[int, str]:
     if isinstance(task_id, int):
         return task_id
-    if isinstance(task_id, str) and "-" in task_id:
```

Comment:
This is needed since the new materials document converts all the ints into strings.

Reply:
I'll implement and ID class to fix this.For now this looks good.